docs/modules/knowledge-graph/pages/index.adoc

```diff
@@ -0,0 +1,44 @@
+= Graph-based Knowledge Extraction and Traversal
+
+RAGStack offers two libraries supporting knowledge graph extraction and traversal, `ragstack-ai-knowledge-graph` and `ragstack-ai-knowledge-store`.
+
+A knowledge graph represents distinct entities (real world objects, like "Marie Curie" or "maize") as **nodes**. Nodes are connected by **edges** indicating relationships between them. Each edge includes the source (for example, "Marie Curie" the person), the target ("Nobel Prize" the award) and a type, indicating how the source relates to the target (for example, “won”).
+
+A graph database isn't required to use the knowledge graph libraries - RAGStack uses Astra DB or Apache Cassandra to store and retrieve graphs.
+
+*TL;DR:*
+
+. Retrieval augmented generation based on chunking and vector similarity search has some weaknesses.
+.. Similarity search looks for information most similar to the question. This makes is harder to answer questions with multiple topics.
```

Comment:
Another way of explaining this that I've used:

1. Similarity search only finds similar information.
2. There are many cases where less-similar information is relevant to the question. Consider a tech support system where you reach an article which *is* similar, that says "if you have trouble with step 4, see <blah>". Even if `<blah>` is *not* similar to the question, it likely provides more information -- the author even says "see also". This is an easy edge to understand (from the coarse grained graph case)
3. Alternatively, if you retrieve 100 chunks that are all highly similar to the question, you have likely retrieved 100 chunks that are also highly similar to themselves, even if you employ MMR. Following edges to "linked" information can allow you to increase the diversity of the results, which provides more distinct information (when combined).